tests/cortexm_common_ldscript/Makefile

```diff
@@ -86,10 +86,15 @@ OFFSETS_TESTS = 0x1000 0x2000
 tests-offsets: $(OFFSETS_TESTS:%=test-offset_%)
 
 .PHONY: test-offset_%
+# Match the 'code' section. It is usually '.text' but is '.vector' on `kinetis`.
+# * [ 1] .text             PROGBITS        08001000 001000 001ef8 ...
+# * [ 1] .vector           PROGBITS        00001000 001000 000400 ...
+# So match with the section [ 1] being `PROGBITS`. Adapt if new cases appear.
+CODE_SECTION = \[ 1\] \.[a-zA-Z_.]* *PROGBITS
```

Comment:
`[a-zA-Z_.]` Do sections have "." in the middle of their names? you are already matching the on leading the section name.

Reply:
I used the output of for a node as example:

And `.ARM.attributes` has one.

```
Section Headers:
  [Nr] Name              Type            Addr     Off    Size   ES Flg Lk Inf Al
  [ 0]                   NULL            00000000 000000 000000 00      0   0  0
  [ 1] .text             PROGBITS        08001000 001000 001ed0 00  AX  0   0  4
  [ 2] .stack            NOBITS          20000000 020000 000200 00  WA  0   0  1
readelf: Warning: [ 3]: Link field (0) should index a symtab section.
  [ 3] .relocate         REL             20000200 010200 000078 08  WA  0   0  4
  [ 4] .noinit           PROGBITS        20000278 010278 000000 00   W  0   0  1
  [ 5] .bss              NOBITS          20000278 010278 0007fc 00  WA  0   0  4
  [ 6] .debug_info       PROGBITS        00000000 010278 016389 00      0   0  1
  [ 7] .debug_abbrev     PROGBITS        00000000 026601 00378e 00      0   0  1
  [ 8] .debug_aranges    PROGBITS        00000000 029d8f 000558 00      0   0  1
  [ 9] .debug_ranges     PROGBITS        00000000 02a2e7 0005e8 00      0   0  1
  [10] .debug_macro      PROGBITS        00000000 02a8cf 01e43a 00      0   0  1
  [11] .debug_line       PROGBITS        00000000 048d09 009a70 00      0   0  1
  [12] .debug_str        PROGBITS        00000000 052779 03e32e 01  MS  0   0  1
  [13] .comment          PROGBITS        00000000 090aa7 00007f 01  MS  0   0  1
  [14] .ARM.attributes   ARM_ATTRIBUTES  00000000 090b26 000033 00      0   0  1
  [15] .debug_frame      PROGBITS        00000000 090b5c 00105c 00      0   0  4
  [16] .debug_loc        PROGBITS        00000000 091bb8 001a2d 00      0   0  1
  [17] .symtab           SYMTAB          00000000 0935e8 001d90 10     18 271  4
  [18] .strtab           STRTAB          00000000 095378 000c0a 00      0   0  1
  [19] .shstrtab         STRTAB          00000000 095f82 0000cb 00      0   0  1
```

Reply:
Perfect :)